modules/ROOT/pages/datamapper-concepts.adoc

```diff
@@ -125,13 +125,13 @@ As you can see, the output is a large and complex structure with nested items at
 
 .. Click the *User Defined* radio button, then click *Create/Edit Structure*
 +
-image::input-and-output.jpg[input+and+output]
+image::input-and-output.jpg["A data mapping tool with the capability to create a mapping from a list of statuses to a CSV file"]
```

Comment:
Does this revision make sense:

"A data mapping tool that converts a map of a status list into a CSV file"